pkg/utils/config/config.go

```diff
@@ -72,15 +73,39 @@ type Filter struct {
 	Exclude []string `yaml:"exclude" mapstructure:"exclude"`
 }
 
+// Images is the image configuration.
+type Images struct {
+	ImageFilter `yaml:",inline" mapstructure:",inline"`
+}
+
+// ImageFilter is the image filter configuration.
+type ImageFilter struct {
+	// ExcludeImages is a regex that must match in order for an image to be excluded and not pinned
+	ExcludeImages []string `yaml:"exclude_images" mapstructure:"exclude_images"`
+	ExcludeTags   []string `yaml:"exclude_tags" mapstructure:"exclude_tags"`
+}
+
 // ParseConfigFile parses a configuration file.
 func ParseConfigFile(configfile string) (*Config, error) {
 	bfs := osfs.New(".")
 	return ParseConfigFileFromFS(bfs, configfile)
 }
 
+func defaultConfig() *Config {
+	return &Config{
+		Platform: "linux/amd64",
```

Comment:
Should we rather default to no platform set?

Reply:
we should IMO